generators/app/index.ts

```diff
@@ -90,6 +90,7 @@ module.exports = class extends Generator {
         prettier: "^2.x",
         "prettier-plugin-apex": "^1.x",
         "pretty-quick": "^3.x",
+        "sfdx-cli": "^7.106.3",
```

Comment:
this will install the latest version of the `sfdx-cli`.  The `^` allows for minor version changes (eg 7.x.x)